extensions/wikia/ContentReview/controllers/templates/ContentReviewSpecialController_index.php

```diff
@@ -36,7 +36,7 @@
 						<?= wfMessage( 'content-review-special-review-open' )->escaped() ?>
 					<? else: ?>
 						<a href="<?= Sanitizer::encodeAttribute( $review['diff'] ) ?>" target="_blank"
```

Comment:
No reason to decide for user to open link in new tab. Especially that opening in same window doesn't break anything.